x-pack/plugins/apm/public/components/shared/TruncateHeightSection/index.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+import { findDOMNode } from 'react-dom';
```

Comment:
`findDOMNode` has been deprecated in strict mode, so perhaps we should avoid it?
https://reactjs.org/docs/react-dom.html#finddomnode